[2021-10-29 18:56:20,728] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: first_python_dag.print_the_context manual__2021-10-29T18:56:19.149512+00:00 [queued]>
[2021-10-29 18:56:20,731] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: first_python_dag.print_the_context manual__2021-10-29T18:56:19.149512+00:00 [queued]>
[2021-10-29 18:56:20,732] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-10-29 18:56:20,733] {taskinstance.py:1242} INFO - Starting attempt 1 of 1
[2021-10-29 18:56:20,733] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-10-29 18:56:20,738] {taskinstance.py:1262} INFO - Executing <Task(PythonOperator): print_the_context> on 2021-10-29 18:56:19.149512+00:00
[2021-10-29 18:56:20,740] {standard_task_runner.py:52} INFO - Started process 80 to run task
[2021-10-29 18:56:20,742] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'first_python_dag', 'print_the_context', 'manual__2021-10-29T18:56:19.149512+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/first_python_dag.py', '--cfg-path', '/tmp/tmp_jq053j3', '--error-file', '/tmp/tmph54ubwp4']
[2021-10-29 18:56:20,743] {standard_task_runner.py:77} INFO - Job 4: Subtask print_the_context
[2021-10-29 18:56:20,757] {logging_mixin.py:109} INFO - Running <TaskInstance: first_python_dag.print_the_context manual__2021-10-29T18:56:19.149512+00:00 [running]> on host 49d47df7fb89
[2021-10-29 18:56:20,770] {taskinstance.py:1412} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=first_python_dag
AIRFLOW_CTX_TASK_ID=print_the_context
AIRFLOW_CTX_EXECUTION_DATE=2021-10-29T18:56:19.149512+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-10-29T18:56:19.149512+00:00
[2021-10-29 18:56:20,968] {logging_mixin.py:109} INFO - 1.3.4
[2021-10-29 18:56:20,969] {python.py:152} INFO - Done. Returned value was: 1
[2021-10-29 18:56:20,978] {taskinstance.py:1270} INFO - Marking task as SUCCESS. dag_id=first_python_dag, task_id=print_the_context, execution_date=20211029T185619, start_date=20211029T185620, end_date=20211029T185620
[2021-10-29 18:56:20,996] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-10-29 18:56:21,003] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
